### ISSUE TITLE
cbp2make required for Linux build (master branch)?

### ISSUE THREAD

**Issue by m00dawg** [smeighan/xLights#661]
Is cbp2make a requirement for building on Linux moving forward? I've gotten stuck with needing to build it, which is a new requirement (seems to have been introduced within this past week). Docs don't mention anything about it, so I wanted to make sure that is something to do moving forward (if so, I can submit a PR to update the docs accordingly).

**Comment by AzGilrock** [comment]
I don't think so but someone has to update the Linux makefile when we add files.  I think I've done it manually or you can use that.  If you try to build before Chris updates it then yeah you gotta do extra work.

**Comment by keithsw1111** [comment]
If you want to build on Linux the only source you can be sure that works is the ones Chris checks in.

**Comment by AzGilrock** [comment]
The builds aren't all synchronized.  The Windows and Mac usually release together then Chris updates Linux.

**Comment by m00dawg** [comment]
Gotcha, thanks for the info folks!

**Comment by stampedeboss** [comment]
Having just built 46 on Linux, yes it's needed.

**Comment by m00dawg** [comment]
The cbp2make docs were a bit fuzzy on how to build it, but once I figured that out, installing that was easy and yep I was able to build xLights on Linux just fine. The steps were basically:

1. Download cbp2make source
2. Unpack and symlink cbp2make.cbp.mak.unix to Makefile
3. Run make
4. Copy bin/Release/cbp2make to /usr/local/bin

After that I was able to build xLights by just doing a 'make clean && make -j4'
